docker/docker-compose.yml

```diff
@@ -0,0 +1,73 @@
+version: "3.8"
+
+# Naming the project allows us to use it when deleting containers so we can target only ones
+# generated by this project
+# https://docs.docker.com/compose/environment-variables/envvars/#compose_project_name
+name: cuttle
+
+# Share volumes between images, see https://stackoverflow.com/a/62873977
+#  - https://docs.docker.com/compose/compose-file/07-volumes/
+#  - https://docs.docker.com/compose/compose-file/compose-file-v3/#extension-fields
+x-app-volume:
```

Comment:
This allows a top level config for multiple services to share. In this case the cuttle src directory.

This may not be enough since it won't watch any root file changes like the vite config but I think that's acceptable for now.